### PR TITLE
[Modular] Aphrodisiacs

### PR DESCRIPTION
**Pull request by linnpap**
## About The Pull Request

Adds crocin and hexacrocin, along with their respective kegs, back into the game.

## Why It's Good For The Game

Used to be in the oldbase.

## Changelog
:cl:Linnepap
add: Adds crocin and hexacrocin
add: Adds pref to enable/disable being affected by aphrodisiacs
/:cl:

### CODE REVIEW

**Comment by H0gberg** [comment]
YES YES YES YES YES YES YSE SEYEYSYEF ISEUFGHIS HFS 

### _**SPEED MERGE**_

**Comment by MLGTASTICa** [comment]
No , they are only used for pref break and have lead to way too many issues.

**Comment by desertmote** [comment]
> No , they are only used for pref break and have lead to way too many issues.

add a check that makes sure nothing happens if non-con is on NO.

**Comment by Vynellin** [comment]
> > No , they are only used for pref break and have lead to way too many issues.
> 
> add a check that makes sure nothing happens if non-con is on NO.

Hmm yes im just sitting in the bar I want to be Aphrodisiac bombed for the third time

**Comment by Low3** [comment]
Just ban people that use it for prefbreak/make it not work on people that have them disabled in prefs.

**Comment by linnpap** [comment]
> 
> 
> Just ban people that use it for prefbreak/make it not work on people that have them disabled in prefs.

i made sure that that theres a pref for disabling/enabling being affected by aphros in it

**Comment by Low3** [comment]
Tbh i think lewd chems need different pref setting. Someone might be into non-con but against chems and vice versa.

**Comment by linnpap** [comment]
> 
> 
> Tbh i think lewd chems need different pref setting. Someone might be into non-con but against chems and vice versa.

thats exactly what i already wrote in there

**Comment by Low3** [comment]
> > Tbh i think lewd chems need different pref setting. Someone might be into non-con but against chems and vice versa.
> 
> thats exactly what i already wrote in there

Then its epic. Should probably add that in description/changelog.

File: modular_skyrat/modules/aphros/readme.dm [hunk 1]
```diff
@@ -0,0 +1,36 @@
+https://github.com/Skyrat-SS13/Skyrat-tg/pull/2914
+
+## Title: Aphrodisiacs
+
+MODULE ID: N/A
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/aphros/readme.dm]:
```suggestion
MODULE ID: APHROS
```

File: modular_skyrat/modules/aphros/readme.dm [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+### Description:
+
+Adds aphrodisiacs back into the game, along with their respective kegs and client prefs.
+
+### TG Proc/File Changes:
+
+- modular_skyrat\modules\customization\modules\client\preferences.dm
+<!-- If you had to edit, or append to any core procs in the process of making this PR, list them here. APPEND: Also, please include any files that you've changed. .DM files that is. -->
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/aphros/readme.dm]:
```suggestion
```

File: modular_skyrat/modules/aphros/readme.dm [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+### Defines:
+
+- code\__DEFINES\~skyrat_defines\preferences.dm
+<!-- If you needed to add any defines, mention the files you added those defines in -->
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/aphros/readme.dm]:
```suggestion
```

File: modular_skyrat/modules/aphros/readme.dm [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+### Master file additions
+
+- N/A
+<!-- Any master file changes you've made to existing master files or if you've added a new master file. Please mark either as #NEW or #CHANGE -->
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/aphros/readme.dm]:
```suggestion
```

File: modular_skyrat/modules/aphros/readme.dm [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+### Included files that are not contained in this module:
+
+- N/A
+<!-- Likewise, be it a non-modular file or a modular one that's not contained within the folder belonging to this specific module, it should be mentioned here -->
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/aphros/readme.dm]:
```suggestion
```

**Comment by Oscari-pog** [comment]
Me when I ask for a drink in bar and suddenly I start getting flavor messages about being horny

(if its a checkmark to prevent that then gucci)

**Comment by Gandalf2k15** [comment]
Fix conflicts and we can TM!

**Comment by CameronWoof** [comment]
What if we didn't go back down the road of mechanical NRP sex shit? We JUST got rid of all of it. Why do this now?

**Comment by Ducky-Momo** [comment]
honestly considering we've got the option to disable it i'm all for this

**Pull request merged**